src/reaper_osara.cpp

```diff
@@ -2509,11 +2509,16 @@ map<pair<int, int>, Command*> commandsMap;
 
 const char CONFIG_SECTION[] = "osara";
 
+extern bool shouldReportNotes;
+extern bool shouldPreviewNotes;
```

Comment:
I'd ideally prefer these options were split into a separate PR, but it's not a big deal and it's probably a bit of work to split it out now, so I'll live with it. :)

I can understand the desire to turn off note reporting. That way, you can use speech for note position and audio for the notes themselves, increasing efficiency. However, I'm not convinced we have a use case for disabling preview? Configurability can be a good thing, but only if every single option adds real value. Otherwise, it adds unnecessary UI complexity.

I'm also concerned about the possibility of someone disabling both note preview and note reporting. That seems like something the UI shouldn't allow. We'd have to use a radio button to avoid that, though: Indicate midi notes by: reporting/previewing/both... or similar.

Reply:
I think you're true. I also noticed that Reaper also has its own note previewing in some cases, having Osara disable note preview and then Reaper preview notes would be confusing.
I removed the option for now. I'm a big fan of disabling reporting, though.